js/main.js

```diff
@@ -11,14 +11,19 @@ Site = {
     });
 
     $(document).ready(function () {
-
+      if ($('body').hasClass('page-info')) {
+        _this.Info.init();
+      }
     });
 
   },
 
   onResize: function() {
     var _this = this;
 
+    if (_this.Info.$team.length) {
```

Comment:
I think this functionality should be in a function on the Info object that is called from this Site resize event.